.github/workflows/integration.yml

```diff
@@ -16,7 +16,7 @@ name: Integration Test
 
 env:
     CONFIG_FILE: deploy.toml
-    IMAGE_NAME: $(echo "ghcr.io/${{ github.repository }}/echo" | tr '[:upper:]' '[:lower:]')
+    IMAGE_NAME: $(echo "192.168.50.2:5000/echo" | tr '[:upper:]' '[:lower:]')
```

Comment:
Can we replace this with just `IMAGE_NAME: 192.168.2:5000/echo`?

Reply:
Good point, done!